src/services/auth.services.js

```diff
@@ -0,0 +1,62 @@
+const { ApiError } = require('../exeptions/auth.error');
+const { User } = require('../models/user');
+const { hashPassword } = require('../utils/password/hashPassword');
+const { randomUUID } = require('crypto');
+
+const signUp = async ({ email, password }) => {
+  const hashedPassword = await hashPassword(password);
+
+  const user = await User.create({
+    email,
+    password: hashedPassword,
+    activationToken: randomUUID(),
+  });
+
+  return user;
+};
+
+const activate = async (activationToken) => {
+  const user = await User.findOne({ where: { activationToken } });
+
+  if (!user) {
+    throw ApiError.badRequest('Bad activation link');
```

Comment:
The `ApiError` constructor arguments are incorrectly ordered. Ensure the correct order is `message, status, errors`.